share/governance.md

```diff
@@ -0,0 +1,111 @@
+
+*How to become one*: Submit a pull request to a Vox Pupuli project
+
+
+Contributors
+------------
+
+
+Contributors are collaborators who have shown wide dedication to the Vox Pupuli project in general or deep dedication to one project in particular, and the ability to work well with collaborators, users, and other contributors. The contributors have responsibilities beyond the collaborators. In particular, contributors formally decide on whether a pull request is merged or a release is cut. Contributors are recognized and valued for their contributions, in whatever form those contributions take. Tasks such as communication, documentation, art, packaging, and testing are all valued contributions. A contributor will use lazy consensus to decide on whether to merge a pull request from a collaborator. If the discussion is no longer moving towards a consensus, the PMC must vote via lazy consensus on whether the patch should be applied.
+
+*How to become one*: Be a collaborator and be nominated to the PMC as a contributor. Nominations should be sent to <email address for nominations>. You may nominate yourself. It is also common to give contributor status to an individual who donates code to the project by migrating a repository to the github namespace.
+
+
+Project Management Committee (PMC)
+----------------------------------
+
+
+The project management committee has responsibilities beyond contributors that include participating in strategic planning, release planning, and approving changes to the governance model. They also have the duty of upholding the community code of conduct(https://voxpupuli.org/coc/) and ensuring its values. The PMC is also responsible for updating the community code of conduct. It also makes decisions when community consensus cannot be reached. 
+
+The PMC has final say over who can become a committer and will use lazy consensus for approval. Discussion over committer nominations will be done in private.
+
+Membership of the PMC is by election. Condorcet(https://en.wikipedia.org/wiki/Condorcet_method) voting is held once a year. A non-running election official is appointed by the PMC and runs the election. There are 5 members of the PMC. Terms are 1 year.
+
+The PMC doesn't have specific roles or a chairperson.
+
+
+*How to become one*: Win in the yearly election.
+
+
+
+Specialized Roles
+-----------------
+
+
+There are three officer roles in VoxPupuli. They can all be the same person, as long as the work is getting done. They can have any other role in the organization, including being on the PMC. The PMC is responsible for appointments, and appointments last until either they are removed or the officer steps down. The PMC can add more officer roles as it sees fit.
+
+1) Election Officer (the election officer can be on the PMC, as long as they are not running in the election they are officiating)
```

Comment:
Formatting: These definitions are not parenthetical. Use `:` instead. Example:

`Foo: The Foo does bar when baz.`